### DISCUSSION TITLE
Workflow file to generate windows Image

### DISCUSSION THREAD

**Discussion by suhas-arcadis** [actions/virtual-environments#6016] [General questions]
Anyone has an example of windows build github action yml? I would like to build the image using github actions. Also, is there a single powershell script which I can use to install all the dependent tools required for a windows runner to be hosted as self hosted runner on Azure..

**Comment by mikhailkoliada** [comment]
Hello @suhas-arcadis, I believe https://github.com/orgs/community/discussions/ is the best place to find answer to your question
